tests/page/elementhandle-screenshot.spec.ts

```diff
@@ -168,7 +168,8 @@ it.describe('element screenshot', () => {
 
   it('should work with a rotated element', async ({ page }) => {
     await page.setViewportSize({ width: 500, height: 500 });
-    await page.setContent(`<div style="position:absolute;
+    await page.setContent(`<!DOCTYPE html>
```

Comment:
We should not need to modify tests to make them pass. If Firefox behaves differently w/o doctype we should fix it elsewhere (prepend DOCTYPE when settings content or differently)

Reply:
ok, I've removed the doctype update from this PR

Reply:
I'd be happy to help, but I don't understand what this change in a test file has to do with the quirks mode. Looks like it's not required for generating the screenshot expectations, so let me merge this PR and we can discuss the test change in a separate issue/PR.

Reply:
I've created the issue for it: https://github.com/microsoft/playwright/issues/34499.